config/src/window.rs

```diff
@@ -0,0 +1,26 @@
+use wezterm_dynamic::{ToDynamic, FromDynamic};
+
+
+#[derive(Debug, Clone, ToDynamic, PartialEq, Eq, FromDynamic)]
+pub enum WindowLevel {
+    AlwaysOnBottom = -1,
+    Normal = 0,
+    AlwaysOnTop = 3,
+}
+
+impl Default for WindowLevel {
+    fn default() -> Self {
+        WindowLevel::Normal
+    }
+}
```

Comment:
You can reduce the verbosity of this by changing how you declare the enum, and having rust derive it for you.

Also, I just noticed the integer values here; let's remove them as they are also a macos implementation detail that shouldn't bleed into the config crate.

```suggestion
#[derive(Default, Debug, Clone, ToDynamic, PartialEq, Eq, FromDynamic)]
pub enum WindowLevel {
    AlwaysOnBottom,
    #[default]
    Normal,
    AlwaysOnTop,
}
```